pkg/network/ovn_kubernetes_test.go

```diff
@@ -262,6 +262,34 @@ func TestOVNKubernetesIsSafe(t *testing.T) {
 	errs := isOVNKubernetesChangeSafe(prev, next)
 	g.Expect(errs).To(BeEmpty())
 
+	// try to add a new hybrid overlay config
+	hybridOverlayConfigNext :=
+		operv1.HybridOverlayConfig{
+			HybridClusterNetwork: []operv1.ClusterNetworkEntry{
+				operv1.ClusterNetworkEntry{CIDR: "00.0.0.0", HostPrefix: 80},
```

Comment:
"00.  should be "0. Is HostPrefix: 80 valid? Also 0.0.0.0 is special, I would start somewhere else.

Reply:
because it is unit tests and just a string it doesnt really matter but changing to CIDR "10.132.0.0/14" and hostsubnet 23